{"name": "test_get_all_roles", "status": "failed", "statusDetails": {"message": "AssertionError: GlobalErrorMessages.WRONG_STATUS_CODE", "trace": "user_data = {'email': 'wcurgq48@email.com', 'employeeId': 97865, 'id': 308, 'isDefaultPassword': True, ...}\n\n    def test_get_all_roles(user_data):\n        e = AuthorisedHeader(user_data)\n        header = e.header\n        response = requests.request('GET', url = auth_url + '/roles', headers=header)\n        response = ResponseValidator(response)\n>       response.assert_status_code(200).validator(AllRoles)\n\ntests/useful_contact/get/test_get_all_roles.py:12: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <base.ResponseValidator object at 0x7f4916550280>, status_code = 200\n\n    def assert_status_code(self,status_code):\n>       assert self.response.status_code == status_code, GlobalErrorMessages.WRONG_STATUS_CODE\nE       AssertionError: GlobalErrorMessages.WRONG_STATUS_CODE\n\nbase.py:67: AssertionError"}, "start": 1652258015055, "stop": 1652258015060, "uuid": "fa6c5c26-20b0-4e5c-a223-21a004b2f66c", "historyId": "5a7f28d3f384ef99a7f5be99f380c5f1", "testCaseId": "a3098fda6afb088402d067446357171d", "fullName": "tests.useful_contact.get.test_get_all_roles#test_get_all_roles", "labels": [{"name": "parentSuite", "value": "tests.useful_contact.get"}, {"name": "suite", "value": "test_get_all_roles"}, {"name": "host", "value": "ubuntu-desktop-i5-7500"}, {"name": "thread", "value": "24449-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.useful_contact.get.test_get_all_roles"}]}